stable/redis-ha/README.md

```diff
@@ -58,28 +58,28 @@ $ helm install \
-| `servers.annotations`            | See Appliance mode                                                                                                           | ``                                                        |
-| `rbac.create`                    |  whether RBAC resources should be created                                                                                    | true                                                      |
-| `serviceAccount.create`          | whether a new service account name that the agent will use should be created.                                                | true                                                      |
-| `serviceAccount.name`            | service account to be used.  If not set and serviceAccount.create is `true` a name is generated using the fullname template. | ``                                                        |
+| Parameter                        | Description                                           | Default                                                   |
+| -------------------------------- | ----------------------------------------------------- | --------------------------------------------------------- |
+| `redis_image`                    | Redis image                                           | `quay.io/smile/redis:4.0.6r2`                             |
+| `resources.server`               | CPU/Memory for redis nodes resource requests/limits  | Memory: `200Mi`, CPU: `100m`                              |
+| `resources.sentinel`             | CPU/Memory for sentinel node resource requests/limits | Memory: `200Mi`, CPU: `100m`                              |
+| `replicas.servers`               | Number of redis master/slave pods                     | 3                                                         |
+| `replicas.sentinels`             | Number of sentinel pods                               | 3                                                         |
+| `nodeSelector`                   | Node labels for pod assignment                        | {}                                                        |
+| `tolerations`                    | Toleration labels for pod assignment                  | []                                                        |
+| `servers.serviceType`            | Set to "LoadBalancer" to enable access from the VPC   | ClusterIP                                                 |
+| `servers.annotations`            | See Appliance mode                                    | ``                                                        |
+| `serviceAccount.create`                    | If true, create & use RBAC resources                  | `false`                                                   |
+| `max_memory`                     | Sets the maximum memory redis can allocate            | "" This defaults to the maximum memory allowed               |
```

Comment:
- `max_memory` is out of line with the table
- Also, values keys should be lower camel case (`maxMemory`). See [Chart Best Practices > Naming Conventions](https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/values.md#naming-conventions).
    - I see this chart already had a mix of values key case patterns (for example, there is both `redis_image` and `nodeSelector`). You could fix that in this PR as well (change `redis_image` to `redisImage` or just `image`).